### ISSUE TITLE
run node cpu-load experiment,result is 'can not find the scope controller for creating'

### ISSUE THREAD

**Issue by lcwus2** [chaosblade-io/chaosblade-operator#44]
## Issue Description

 *bug report*

### Describe what happened (or what feature you want)
I'd like to run node-cpu-load experiment, but it returned error

### How to reproduce it (as minimally and precisely as possible)

1. 
2. 
3. 

### Tell us your environment
cpu-load.yaml：

```
apiVersion: chaosblade.io/v1alpha1
kind: ChaosBlade
metadata:
  name: cpu-load
spec:
  experiments:
  - scope: node
    target: cpu
    action: fullload
    desc: "increase node cpu load by names"
    matchers:
    - name: names
      value:
      - "10.23.69.180"
    - name: cpu-percent
      value:
      - "80"
```
![image](https://user-images.githubusercontent.com/30100640/96703849-ebc56780-13c5-11eb-93b7-4af478a3acc8.png)

**Comment by tiny-x** [comment]
Node experiment support SSH channels only, view use cases
````
blade create k8s node-cpu load -h
````

**Comment by lcwus2** [comment]
I see, but this only allow ssh connection without password?
Now I want to test a node in public cloud，but I can't ssh it

**Comment by tiny-x** [comment]
After execution, enter the password interactively.

SSH key login is also supported. You need to compile it or you can search for DingTalk group 23177705 to join it.
https://github.com/chaosblade-io/chaosblade-exec-os/commit/b0a9a54cce591d308898adc69a30c492e4398440

**Comment by tiny-x** [comment]
The problem has been solved in the version 0.8.0
